content/en/opentelemetry/interoperability/instrumentation_libraries.md

```diff
@@ -36,7 +36,7 @@ Datadog instrumentation to avoid duplicate spans in the trace.
 | Java     | v1.35.0                  |
 | Python   | v2.10.0                  |
 | Ruby     | v2.1.0                   |
-| Go       | TBA                      |
+| Go       | v1.67.0                  |
```

Comment:
We don't have the 1.67 version yet, so it might be misleading for another month or so

Reply:
Would it make sense to say v1.65.0 (it has everything except the Span Events APIs) and then once v1.67.0 lands we can update this?

Reply:
Closing this. In Stuart's main PR we simply entered v1.65.0